fr_FR/LC_MESSAGES/theatre.po

```diff
@@ -415,10 +415,10 @@ msgid "GIFT_ERROR_INVALID_CODE"
 msgstr "Ceci n'est pas un code de ticket valide. S'il vous plaît, vérifier pour des typographies et réessayer."
 
 msgid "GIFT_ERROR_INVALID_FORMAT"
-msgstr "Please enter a valid 12-digit ticket number."
+msgstr "Veuillez s'il vous plaît."
```

Comment:
Hi, this sentence is not complete, there should be something like "Veuillez entrer un numéro de ticket à 12 chiffres valide".

btw thank you for your contributions!

Reply:
Oh yeah lmao, thx for review, don't know how that passed me